tests/integration/api/test_activation_with_webhook.py

```diff
@@ -246,6 +246,30 @@ def test_create_activation_with_webhooks(
     assert data["webhooks"][0]["name"] == "demo"
 
 
+@pytest.mark.django_db
+def test_list_activations_by_webhook(
+    admin_client: APIClient,
+    default_activation: models.Activation,
+    new_activation: models.Activation,
+    default_webhook: models.Webhook,
+):
+    activation_1 = default_activation
+    activation_2 = new_activation
+
+    activation_1.webhooks.add(default_webhook)
```

Comment:
@hsong-rh Can we add a test for no activations connected to EventStreams

Reply:
Added